### ISSUE TITLE
Set up a configurable throttling mechanism in loading pipeline if necessary

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/google-books#2]
Since we will be waiting on Project CHAOS and disk space until we can fully enable the load pipeline, we need to ability to throttle the load to a level that allows to keep books moving and exercising the pipeline while not stressing the limits of our storage capacity.

**Comment by jcoyne** [comment]
Superseded by #77
